tests/cases/buffer_load_binary.amber

```diff
@@ -0,0 +1,55 @@
+#!amber
+#
+# Copyright 2020 The Amber Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SHADER compute compute_shader GLSL
+#version 430
+
+layout(set = 0, binding = 0) buffer block0 {
+  float in_data[12];
+};
+
+layout(set = 0, binding = 1) buffer block1 {
+  float out_data[12];
+};
+
+void main() {
+    for (int i = 0; i < 12; i++)
+        out_data[i] = in_data[i] * 2.0;
+}
+END
+
+BUFFER buf0 DATA_TYPE vec4<float> SIZE 3 FILE vec4data.bin
```

Comment:
Looking at the test case, I'm not sure if binary is what we want here. The `vec4data.bin` file seems like it would be much more useful if it was actually text? So, then I could look at it and edit it? Right now, how do you create/edit one of these binary files?